src/lio/bin/lio_mv.c

```diff
@@ -194,13 +197,15 @@ int main(int argc, char **argv)
         } else {
             info_printf(lio_ifd, 0, "ERROR: Multiple paths selected but the dest(%s) isn't a directory!\n", dtuple.path);
         }
+        n_errors = 1;
```

Comment:
OK. But what is `n_errors` supposed to be? Up top, it's the counter of the errors that happened. At lines 289 and 300, you add up the error codes for any child GOPs, then you return it to the caller. Is it the number of errors? Is it "Sum of error codes" + "number of errors"? What does that number mean? Are there some sort of semantics to the exit code a user is supposed to interpret?